SPEC.md

```diff
@@ -323,6 +328,20 @@ An example trace:
 
 ```
 
+### Measurements
+
+Measurements are a separate kind of document that engines can emit. Measurements
+are a numeric value to a given named key. Measurements are primarily used for
+tracking rates of certain kinds of errors engines may encounter internally (e.g.
+parse errors).
```

Comment:
What do you think of generalizing this description a bit?

Maybe something like...

> Measurements are a numeric value bound to a named key. Measurements may represent a variety of internal engine metrics. For example, an engine may emit a measurement representing the rate of parsing errors during an analysis.

Reply:
Updated. These documents are not always going to be for "internal engine metrics", though: that's just our first intended use of them. I rephrased that sentence a bit.